packages/desktop-client/src/components/sidebar/Account.tsx

```diff
@@ -140,7 +146,21 @@ export function Account({
             </View>
 
             <AlignedText
-              left={name}
+              left={
+                !!account?.id ? (
+                  <Tooltip
+                    content={<Notes notes={note} />}
```

Comment:
Can we only render the actual notes in the `Notes` component and render the account name in a separate component?

Reply:
We could. I did that to use the Markdown to bold the name, instead of CSS.